[2024-12-05T20:26:59.261+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-05T20:26:59.276+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT_dbt.dbt_test manual__2024-12-05T20:26:50.679649+00:00 [queued]>
[2024-12-05T20:26:59.280+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT_dbt.dbt_test manual__2024-12-05T20:26:50.679649+00:00 [queued]>
[2024-12-05T20:26:59.281+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-05T20:26:59.287+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_test> on 2024-12-05 20:26:50.679649+00:00
[2024-12-05T20:26:59.290+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ELT_dbt', 'dbt_test', 'manual__2024-12-05T20:26:50.679649+00:00', '--job-id', '508', '--raw', '--subdir', 'DAGS_FOLDER/ELT_metrics.py', '--cfg-path', '/tmp/tmpq6dw5i6h']
[2024-12-05T20:26:59.292+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=565) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-05T20:26:59.292+0000] {standard_task_runner.py:91} INFO - Job 508: Subtask dbt_test
[2024-12-05T20:26:59.292+0000] {standard_task_runner.py:63} INFO - Started process 566 to run task
[2024-12-05T20:26:59.315+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT_dbt.dbt_test manual__2024-12-05T20:26:50.679649+00:00 [running]> on host bd7c71f68e16
[2024-12-05T20:26:59.355+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T20:26:50.679649+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T20:26:50.679649+00:00'
[2024-12-05T20:26:59.356+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-05T20:26:59.363+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-05T20:26:59.363+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/dbt_proj --project-dir /opt/***/dbt_proj']
[2024-12-05T20:26:59.368+0000] {subprocess.py:86} INFO - Output:
[2024-12-05T20:27:00.464+0000] {subprocess.py:93} INFO - [0m20:27:00  Running with dbt=1.8.7
[2024-12-05T20:27:00.975+0000] {subprocess.py:93} INFO - [0m20:27:00  Registered adapter: snowflake=1.8.4
[2024-12-05T20:27:01.209+0000] {subprocess.py:93} INFO - [0m20:27:01  Found 4 models, 3 snapshots, 4 data tests, 5 sources, 459 macros
[2024-12-05T20:27:01.211+0000] {subprocess.py:93} INFO - [0m20:27:01
[2024-12-05T20:27:02.403+0000] {subprocess.py:93} INFO - [0m20:27:02  Concurrency: 1 threads (target='dev')
[2024-12-05T20:27:02.403+0000] {subprocess.py:93} INFO - [0m20:27:02
[2024-12-05T20:27:02.410+0000] {subprocess.py:93} INFO - [0m20:27:02  1 of 4 START test not_null_citibike_historical_summary_month ................... [RUN]
[2024-12-05T20:27:03.007+0000] {subprocess.py:93} INFO - [0m20:27:03  1 of 4 PASS not_null_citibike_historical_summary_month ......................... [[32mPASS[0m in 0.60s]
[2024-12-05T20:27:03.010+0000] {subprocess.py:93} INFO - [0m20:27:03  2 of 4 START test not_null_citibike_historical_summary_year .................... [RUN]
[2024-12-05T20:27:03.807+0000] {subprocess.py:93} INFO - [0m20:27:03  2 of 4 PASS not_null_citibike_historical_summary_year .......................... [[32mPASS[0m in 0.80s]
[2024-12-05T20:27:03.809+0000] {subprocess.py:93} INFO - [0m20:27:03  3 of 4 START test not_null_weather_current_summary_comfort_index ............... [RUN]
[2024-12-05T20:27:04.460+0000] {subprocess.py:93} INFO - [0m20:27:04  3 of 4 PASS not_null_weather_current_summary_comfort_index ..................... [[32mPASS[0m in 0.65s]
[2024-12-05T20:27:04.463+0000] {subprocess.py:93} INFO - [0m20:27:04  4 of 4 START test not_null_weather_current_summary_date ........................ [RUN]
[2024-12-05T20:27:05.293+0000] {subprocess.py:93} INFO - [0m20:27:05  4 of 4 PASS not_null_weather_current_summary_date .............................. [[32mPASS[0m in 0.83s]
[2024-12-05T20:27:05.295+0000] {subprocess.py:93} INFO - [0m20:27:05
[2024-12-05T20:27:05.296+0000] {subprocess.py:93} INFO - [0m20:27:05  Finished running 4 data tests in 0 hours 0 minutes and 4.08 seconds (4.08s).
[2024-12-05T20:27:05.324+0000] {subprocess.py:93} INFO - [0m20:27:05
[2024-12-05T20:27:05.324+0000] {subprocess.py:93} INFO - [0m20:27:05  [32mCompleted successfully[0m
[2024-12-05T20:27:05.324+0000] {subprocess.py:93} INFO - [0m20:27:05
[2024-12-05T20:27:05.325+0000] {subprocess.py:93} INFO - [0m20:27:05  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2024-12-05T20:27:05.902+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-05T20:27:05.903+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-05T20:27:05.917+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT_dbt, task_id=dbt_test, run_id=manual__2024-12-05T20:26:50.679649+00:00, execution_date=20241205T202650, start_date=20241205T202659, end_date=20241205T202705
[2024-12-05T20:27:05.947+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-05T20:27:05.957+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T20:27:05.957+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
